src/components/molecules/footer/BottomIcons.tsx

```diff
@@ -1,8 +1,8 @@
-import BottomNavigation from '@mui/material/BottomNavigation'
+import { BottomNavigation, BottomNavigationAction } from '@mui/material'
 import { RiVimeoFill } from 'react-icons/ri'
 import { FaInstagram } from 'react-icons/fa'
 import { socialLinks } from '../../../assets/socialLinks/SocialLinks'
-import { IconNavigation } from '../../atoms/footer/IconNavigation'
+import { kurtTheme } from '../../../theme'
```

Comment:
🔧 Rather than pull in the theme object (which works?). I believe that best-practive is to use the `useTheme` hook and access the variables from there.

Reply:
I would agree with this in general. We want to use hooks wherever possible, similar to the `useNavigation()` change.